WiitarThing/Components/DeviceControl.xaml.cs

```diff
@@ -763,8 +812,13 @@ private void UpdateIcon(ControllerType cType)
                     break;
 
                 case ControllerType.Drums:
-                    icon.Source = (ImageSource)Application.Current.Resources["DRMIcon"];
-                    UserPrefs.Instance.UpdateDeviceIcon(devicePath, "DRMIcon");
+                    icon.Source = (ImageSource)Application.Current.Resources["WDRIcon"];
+                    UserPrefs.Instance.UpdateDeviceIcon(devicePath, "WDRIcon");
```

Comment:
As mentioned in the App.xaml comment, need `WDRIcon` reverted back to `DRMIcon`.